### ISSUE TITLE
Exchange or feed specific methods should not be in base

### ISSUE THREAD

**Issue by barrysteyn** [snth/numismatic#36]
A good example is CryptoCompare - it has many methods that are specific to itself (for example, `get_historical_hour` or `get_info`. The latter is surprising, but I could not find it on brave new coin.

What we need to do is discuss what specific methods we need (see #35) and move those to the base.

For methods specific, we should add the feed/exchange name as a prefix. For example, `cryptocompare_get_info`. The trick would come in when we want to run this on the cli. So running `coin -f cryptocompare info` would run `cryptocompare_get_info`, and `coin -f bravenewcoin xyz` would run `bravenewcoin_get_xyz`

**Comment by snth** [comment]
I agree with you. We should factor this out into two separate classes, one a low-level `ApiClient` and the other the higher level `DataFeed`.

**Comment by barrysteyn** [comment]
Yes, that would work...
